src/map_object.c

```diff
@@ -1780,7 +1780,7 @@ void sub_02062D10 (MapObject * mapObj)
     MapObject_SetStatusFlagOff(mapObj, (1 << 1));
 }
 
-int sub_02062D1C (const MapObject * mapObj)
+int MapObject_IsMoving(const MapObject *mapObj)
 {
     return sub_020628D8(mapObj, (1 << 1));
```

Comment:
Can we label this routine, if we know that this function is used to determine if a MapObject is moving?

Reply:
Could call it something like `MapObject_GetStateFlag` or `MapObject_GetStatusFlag`. Didn't really know what to go for here